src/Hl7.Fhir.Core/Specification/PocoStructureDefinitionSummaryProvider.cs

```diff
@@ -158,6 +158,8 @@ string getFhirTypeName(Type ft)
 
         public int Order => _pm.Order;
 
+        public string DefaultTypeName => null;
```

Comment:
Is this member HL7v3/CDA specific?  If so, I think we should include that in the name.  If we can find an "equivalent" in FHIR, then we don't need to do that, but then it should not return null but some sensible value when we are dealing with FHIR data.